packages/main/src/StepInput.js

```diff
@@ -186,6 +186,14 @@ const metadata = {
 			defaultValue: 0,
 		},
 
+		/**
+		 * Indicates whether the input is focssed
+		 * @private
+		 */
+		focused: {
+			type: Boolean,
+		},
+
```

Comment:
I could not understand why you add "focused" when you already have a property with the same function called "_focused"

Reply:
done